streampark-console/streampark-console-webapp/src/views/system/variable/Add.vue

```diff
@@ -127,23 +127,21 @@ export default {
     handleVariableCodeBlur (e) {
       const variableCode = (e && e.target.value) || ''
       if (variableCode.length) {
-        if (variableCode.length > 100) {
-          this.validateStatus = 'error'
-          this.help = 'Variable Code should not be longer than 100 characters'
-        } else {
-          this.validateStatus = 'validating'
-          checkVariableCode({
-            variableCode: variableCode
-          }).then((resp) => {
-            if (resp.data) {
-              this.validateStatus = 'success'
-              this.help = ''
-            } else {
-              this.validateStatus = 'error'
-              this.help = 'Sorry, the Variable Code already exists'
-            }
-          })
-        }
+        this.validateStatus = 'validating'
+        checkVariableCode({
+          variableCode: variableCode
+        }).then((resp) => {
+          if (resp.status !== 'success') {
+            this.validateStatus = 'error'
+            this.help = resp.message
```

Comment:
The check can be added in front-end. You can refer how to limit the User name length.